packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -136,7 +136,7 @@ contract MerkleDistributor is Ownable {
      *         when any of individual `_claim`'s `amount` exceeds `remainingAmount` for its window.
      * @param claims array of claims to claim.
      */
-    function claimMulti(Claim[] memory claims) external {
+    function claimMulti(Claim[] memory claims) external virtual {
```

Comment:
Making this `virtual` so I can extend the functionality in `across-smart-contracts-v2`